magpie/api/management/user/user_utils.py

```diff
@@ -119,11 +121,21 @@ def _add_to_group(usr, grp):
 
     user_content = uf.format_user(new_user, new_user_groups)
 
+    # Create a token for the tmp_url, in case an error happens in the webhook services
+    token_id = uuid.uuid4()
+    webhook_token = models.TemporaryToken(
+        token=token_id,
+        operation=TokenOperation.WEBHOOK_ERROR.value,
```

Comment:
I would rename `WEBHOOK_CREATE_USER` to make it distinct from other planned webhooks (on delete, on update)